source/administration/upgrade.rst

```diff
@@ -34,6 +34,62 @@ Owner and group of the install directory - *{owner}* and *{group}*
 
 #. Review the :doc:`important-upgrade-notes` to make sure you are aware of any actions you need to take before or after upgrading from your particular version.
 
+**With a script**

```

Comment:
I feel like the manual upgrade steps should come first before this "with script".

Reply:
I put it there because the script (should) be easier then the manual steps. What do you think @jasonblais?

Reply:
My reason is for the admin to see the manual upgrade first (obviously) and hopefully try it so that the admin may control/feel the entire upgrade process before trying with script.  I think the first part of "with script" - configuration - is key to success of the upgrade process but also easy to make mistake if you're doing the upgrade for the first time.

Reply:
@saturninoabril Completely agree with you here. Doing the change.